src/sp/it/pl/main/JavaLegacy.java

```diff
@@ -11,9 +10,6 @@
 	// requires --add-exports javafx.graphics/com.sun.glass.ui=ALL-UNNAMED
 	@Experimental
 	public static int screenOrdinal(Screen screen) {
-		com.sun.glass.ui.Screen.getScreens().stream()
```

Comment:
you should've removed this in master, has nothing to do with this branch

Reply:
Do we want to adhere to commit purity so rigorously? From where I'm looking, it was safer and more efficient this way.

Reply:
Because it would be more work otherwise and because I would risk potential problems  (like forgetting to remove it, .etc).